src/db/migrations/20250506140752_remove_feature_flags_table.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+export function up(knex) {
+  return knex.schema.dropTableIfExists("feature_flags");
+}
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function down(knex) {
+  return knex.schema.createTable("feature_flags", (table) => {
+    table.string("name").primary().unique();
+    table.boolean("is_enabled").defaultTo(false);
+    table.text("description").defaultTo("");
+    table.specificType("dependencies", "character varying(255)[]");
+    table.specificType("allow_list", "character varying(255)[]");
+    table.specificType("wait_list", "character varying(255)[]");
+    table.timestamp("created_at").defaultTo(knex.fn.now());
+    table.timestamp("modified_at").defaultTo(knex.fn.now());
+    table.timestamp("expired_at");
+    table.timestamp("deleted_at");
+    table.string("owner");
+  });
```

Comment:
This is probably too much effort given that this is so straightforward, but if you want to prevent data loss from a rollback, you could re-insert the feature flags from the `feature_flag_view` with this code: https://github.com/mozilla/blurts-server/blob/680b536055d966ad69fbd0f93e43ec22259199a8/src/db/migrations/20250225111207_add_feature_flag_events.js#L79-L90